dataflow/java/src/main/java/com/httparchive/dataflow/BigQueryImport.java

```diff
@@ -288,20 +318,19 @@ public static void main(String[] args) {
                 DataflowPipelineWorkerPoolOptions.AutoscalingAlgorithmType.NONE);
 
         Pipeline p = Pipeline.create(pipelineOptions);
-
-        PCollectionTuple results = p.apply(TextIO.Read
-                .named("read-har")
-                .from(getHarBucket(options))
-                .withCompressionType(TextIO.CompressionType.GZIP)
-                .withCoder(HarJsonCoder.of()))
+        PCollectionTuple results = p
```

Comment:
I've added decodeHar and unzipGcsFile methods. The project builds successfully now.

However, at run time, I'm getting this error:

> Exception in thread "main" java.lang.IllegalArgumentException: Unable to infer a coder and no Coder was specified. Please set a coder by invoking Create.withCoder() explicitly.
	at com.google.cloud.dataflow.sdk.transforms.Create$Values.apply(Create.java:245)
	at com.google.cloud.dataflow.sdk.transforms.Create$Values.apply(Create.java:213)
	at com.google.cloud.dataflow.sdk.runners.PipelineRunner.apply(PipelineRunner.java:74)
	at com.google.cloud.dataflow.sdk.runners.DirectPipelineRunner.apply(DirectPipelineRunner.java:236)
	at com.google.cloud.dataflow.sdk.Pipeline.applyInternal(Pipeline.java:367)
	at com.google.cloud.dataflow.sdk.Pipeline.applyTransform(Pipeline.java:274)
	at com.google.cloud.dataflow.sdk.values.PBegin.apply(PBegin.java:47)
	at com.google.cloud.dataflow.sdk.Pipeline.apply(Pipeline.java:156)
	at com.httparchive.dataflow.BigQueryImport.main(BigQueryImport.java:340)
Caused by: com.google.cloud.dataflow.sdk.coders.CannotProvideCoderException: Unable to provide a default Coder for com.google.cloud.dataflow.sdk.util.gcsfs.GcsPath. Correct one of the following root causes:
  Building a Coder using a registered CoderFactory failed: Cannot provide coder based on value with class com.google.cloud.dataflow.sdk.util.gcsfs.GcsPath: No CoderFactory has been registered for the class.
  Building a Coder from the @ DefaultCoder annotation failed: Class com.google.cloud.dataflow.sdk.util.gcsfs.GcsPath does not have a @ DefaultCoder annotation.
  Building a Coder from the fallback CoderProvider failed: Cannot provide coder for type com.google.cloud.dataflow.sdk.util.gcsfs.GcsPath: com.google.cloud.dataflow.sdk.coders.Proto2Coder$1@6156496 could not provide a Coder for type com.google.cloud.dataflow.sdk.util.gcsfs.GcsPath: Cannot provide Proto2Coder because com.google.cloud.dataflow.sdk.util.gcsfs.GcsPath is not a subclass of protocol buffer Messsage; com.google.cloud.dataflow.sdk.coders.SerializableCoder$1@3c153a1 could not provide a Coder for type com.google.cloud.dataflow.sdk.util.gcsfs.GcsPath: Cannot provide SerializableCoder because com.google.cloud.dataflow.sdk.util.gcsfs.GcsPath does not implement Serializable.

@kennknowles let me know what you think.